### ISSUE TITLE
bug: install.sh script failing : npm ERR! `user` is not a valid npm option

### ISSUE THREAD

**Issue by nutangwari** [requestly/requestly#785]
### Describe your issue?

OS: windows 11
node: 16.17.1
npm: 9.7.1

`Install.sh` script is failing

`# npm -g config set user root` line in install.sh script at root folder is failing with below error:

`npm ERR! `user` is not a valid npm option`

If I comment out this line, build works correctly, and am able to complete the install.sh script.

### Repro steps

1. Goto project root folder ('requestly')
2. run  `sh install.sh local` script
3. script breaks with error just after npm installation step
4. Now, comment the `#npm -g config set user root`
5. now again run `sh install.sh local` 
6. it works this time.

### Your Environment

Windows 11

### Requestly Version

1.0.3

### Error screenshot

![image](https://github.com/requestly/requestly/assets/9932923/b63da564-f462-416f-8892-bd21757f6afe)

**Comment by lazyvab** [comment]
@nutangwari Thanks for bringing this to our attention. Our development team primarily works on MacOS or Linux based OS and we never encountered this issue. 

I found that this step was actually not needed during installation. I have fixed the script.

Since you are using Windows OS, you might face another issue related to passing environment variables to certain scripts.
Here is a similar thread: https://github.com/requestly/requestly/issues/708#issuecomment-1587257596
There is a workaround suggested which you may try to unblock yourself. 

Closing the issue.
